DisplaySettings/DisplaySettings.json

```diff
@@ -1951,7 +2126,7 @@
             }
         },
         "setVolumeLeveller":{
-            "summary": "Sets the Volume Leveller level. Volume Leveler is an advanced volume-control solution that maintains consistent playback levels for content from different sources.",
+            "summary": "(Version 1)Sets the Volume Leveller level. Volume Leveller is an advanced volume-control solution that maintains consistent playback levels for content from different sources.\n \n### Event \n\n No Events.",
```

Comment:
The use of `(Version 1)` is a new convention and a good idea. Consider updating all RDK services that have methods with multiple versions to use this convention.

Reply:
As discussed keeping only version2 methods in md file.